kubernetes/windows/main.ps1

```diff
@@ -370,8 +434,9 @@ function Read-Configs {
     #Replace placeholders in fluent-bit.conf
     ruby /opt/amalogswindows/scripts/ruby/fluent-bit-conf-customizer.rb
 
-    ruby /opt/amalogswindows/scripts/ruby/tomlparser-geneva-config.rb
-    .\setgenevaconfigenv.ps1
+    # ruby /opt/amalogswindows/scripts/ruby/tomlparser-geneva-config.rb
+    # .\setgenevaconfigenv.ps1
+    Set-ProcessAndMachineEnvVariables "GENEVA_LOGS_INTEGRATION" "false"
```

Comment:
why this being hardcoded to **false**? This going to break the 1p integration, right?

Reply:
This I had before for the issue with the geneva vars. I have fixed this now.